priv/templates/nimble_template/.github/workflows/publish_wiki.yml.eex

```diff
@@ -0,0 +1,51 @@
+name: Publish Wiki
+
+on:
+ push:
+    paths:
+      - .github/wiki/**
+    branches:
+      - develop
+
+env:
+  USER_TOKEN: ${{ secrets.GH_TOKEN }}
+  USER_NAME: github-wiki-workflow
+  USER_EMAIL: ${{ secrets.GH_EMAIL }}
+  OWNER: ${{ github.event.repository.owner.name }}
+  REPOSITORY_NAME: ${{ github.event.repository.name }}
+  WIKI_FOLDER: .github/wiki
+  TEMP_CLONE_WIKI_FOLDER: tmp_wiki
```

Comment:
I've been thinking about this. Should we change this to `TMP_CLONE_WIKI_FOLDER` 💭

Reply:
To be honest, I'd rather keep it as `TEMP_CLONE_WIKI_FOLDER ` as its clearer as a variable name 🤔